.github/workflows/ci.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       matrix:
         # Should match the python versions in pyproject.toml and publish.yml
-        python-version: ["3.10"]
-        os: [ubuntu-latest]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

Comment:
@leng-yue I see you changed this earlier, is there anything I should be aware of? I'll at least try to get things passing like this.

Reply:
Seems there was some issue on 3.12